schemas/macro_service.schema.json

```
@@ -38,9 +39,9 @@
     },
     "HostRequest": {
       "$description": "A request sent from host to macro.",
-      "oneOf": [
-        {"$ref": "#/$defs/AugmentRequest"}
-      ]
+      "properties": {"type": {"type": "string"}, "value": {"oneOf": [
+          {"$ref": "#/$defs/AugmentRequest"}
+      ]}}
```

nit: can we format these so they are easier to understand, for objects with a single key/value or lists with a single entry I am OK with collapsing them on one line, but when it gets nested it gets pretty confusing

```suggestion
      "properties": {
          "type": {"type": "string"},
          "value": {
              "oneOf": [
                  {"$ref": "#/$defs/AugmentRequest"}
              ]
          }
      }
```

---

Yeeeah I was in two minds about this, it's awkward to collapse but it's also awkward expanded as it's boilerplate :)

Switched to auto formatting the whole file, which is probably what we end up doing/enforcing anyway. Thanks.